tests/ci_build/Dockerfile.jvm_cross

```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:18.04
-ARG JDK_VERSION=8
+FROM ubuntu:22.04
+ARG JDK_VERSION=11
```

Comment:
I think this should probably also be in the PR title (assuming that becomes a changelog note). That might be a breaking change for Java users, like anyone using XGBoost4J, right?

Reply:
Changed it back to `8`, to be consistent with
https://github.com/dmlc/xgboost/blob/c9f5fcaf21ea25f78538b672cc60004fce9f524d/tests/buildkite/conftest.sh#L30